sdk/js/src/token_bridge/transfer.ts

```diff
@@ -625,3 +630,154 @@ export async function transferFromAlgorand(
   txs.push({ tx: acTxn, signer: null });
   return txs;
 }
+
+/**
+ * Transfers an asset from Near to a receiver on another chain
+ * @param client
+ * @param coreBridge account 
+ * @param tokenBridge account of the token bridge
+ * @param assetId account
+ * @param qty Quantity to transfer
+ * @param receiver Receiving account
+ * @param chain Reeiving chain
+ * @param fee Transfer fee
+ * @param payload payload for payload3 transfers
+ * @returns [Sequence number of confirmation, emitter]
+ */
+export async function transferTokenFromNear(
+  client: nearAccount,
+  coreBridge: string,
+  tokenBridge: string,
+  assetId: string,
+  qty: bigint,
+  receiver: Uint8Array,
+  chain: ChainId | ChainName,
+  fee: bigint,
+  payload: string = ""
+): Promise<[number, string]> {
+  let wormhole = assetId.endsWith("." + tokenBridge);
```

Comment:
Is this line like calling isWrappedAsset()?

Reply:
yes, probably could call the other function directly